### PR TITLE
Issue13 mms meta

### PR DESCRIPTION
**Pull request by mpejcoch**

### CODE REVIEW

**Review by havardf — APPROVED** [review]
I think this is ok for now.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
+## Build with: docker build -t mmsd .
```
Comment (johtoblan) [inline Dockerfile:1]: Bør forklares i Readme?

File: README.md [hunk 2]
```diff
@@ -7,12 +7,12 @@ Go client for MET Messaging system (MMS)
 
 Shell 1.
 - `cd go-mms`
-- `go build ./cmd/mms-http-server-mock/`
-- `./mms-http-server-mock`
+- `go build ./cmd/rest-api/`
```
Comment (johtoblan) [inline README.md:10]:
```suggestion
- `go build ./cmd/mmsd`
```

File: README.md [hunk 3]
```diff
@@ -7,12 +7,12 @@ Go client for MET Messaging system (MMS)
...
+- `./rest-api`
```
Comment (johtoblan) [inline README.md:11]:
```suggestion
- `./mmsd`
```

**Review by johtoblan — APPROVED** [review]
Looks good, but there seems to be some residue from Hubs -> hub (fixed by  #16 one day )

File: cmd/mms/commands.go [hunk 4]
```diff
@@ -53,6 +53,7 @@ func subscribeEvents(hubs []productionHub) func(*cli.Context) error {
 	}
 }
 
+//TODO: Split in two functions so that it is callable both from cli and the Python binding
```
Comment (vkbo) [inline cmd/mms/commands.go:56]: Don't think this is necessary anymore. I think I will hook directly into `pkg/mms`.

File: templates/howto.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    We will also update the <a href="/docs/changelog">changelog</a>.
+</p>
+<div class="zig-zag"></div>
+
+<h3>Error handling</h3>
+<p>
+    We use regular 4xx and 5xx http status codes, as close as possible to the HTTP specification. Some errors will have a message of what went wrong in it. 
+    Some errors will respons with json. Do not encode those json error responses, as they might change without proper versioning as described above.
+</p>
+    There might be cases where the service can give a successfull response, but the response object is missing some data.
```
Comment (vkbo) [inline templates/howto.html:21]: This text is outside the paragraph.

**Review by vkbo — APPROVED** [review]
Just a minor HTML issue, otherwise I have no comments.

**Pull request merged**